central/processlisteningonport/datastore/datastore_impl_test.go

```diff
@@ -409,2101 +354,2269 @@ func (suite *PLOPDataStoreTestSuite) TestPLOPAddCloseTwice() {
-			expectedPlopDeletions: []string{fixtureconsts.PlopUID1},
-		},
-	}
-	for _, c := range cases {
-		suite.T().Run(c.name, func(t *testing.T) {
-			suite.TearDownTest()
-			suite.SetupTest()
-			// Add deployments if necessary
-			deploymentDS, err := deploymentStore.GetTestPostgresDataStore(suite.T(), suite.postgres.DB)
-			suite.Nil(err)
-			for _, deploymentID := range c.deployments.AsSlice() {
-				suite.NoError(deploymentDS.UpsertDeployment(suite.hasAllCtx, &storage.Deployment{Id: deploymentID, ClusterId: fixtureconsts.Cluster1}))
-			}
-
-			for _, podID := range c.pods.AsSlice() {
-				insertPod := fmt.Sprintf("INSERT INTO pods (id, clusterid) VALUES ('%s', '%s')", podID, fixtureconsts.Cluster1)
-				_, err := suite.postgres.DB.Exec(suite.hasWriteCtx, insertPod)
-				suite.Nil(err)
-			}
-
-			suite.NoError(suite.indicatorDataStore.AddProcessIndicators(suite.hasWriteCtx, c.initialProcesses...))
-			countFromDB, err := suite.indicatorDataStore.Count(suite.hasReadCtx, nil)
-			suite.NoError(err)
-			suite.Equal(len(c.initialProcesses), countFromDB)
-
-			err = suite.store.UpsertMany(suite.hasWriteCtx, c.initialPlops)
-			suite.NoError(err)
-			plopCount, err := suite.store.Count(suite.hasReadCtx, search.EmptyQuery())
-			suite.NoError(err)
-			suite.Equal(len(c.initialPlops), plopCount)
-
-			suite.datastore.PruneOrphanedPLOPsByProcessIndicators(suite.hasAllCtx, orphanWindow)
-
-			plopCount, err = suite.store.Count(suite.hasReadCtx, search.EmptyQuery())
-			suite.NoError(err)
-			suite.Equal(len(c.initialPlops)-len(c.expectedPlopDeletions), plopCount)
-
-		})
-	}
-}
-
-// RemovePLOPsWithoutProcessIndicatorOrProcessInfo Adds a PLOP with a matching process indicator.
-// The process indicator is then deleted. This means that the PLOP has no matching
-// process indicator and does not have any process information. A pruning function
-// which removes such PLOPs is then called.
-func (suite *PLOPDataStoreTestSuite) RemovePLOPsWithoutProcessIndicatorOrProcessInfo() {
-	indicators := getIndicators()
-
-	var indicatorIds []string
-
-	for _, indicator := range indicators {
-		indicatorIds = append(indicatorIds, indicator.Id)
-	}
-
-	plopObjects := []*storage.ProcessListeningOnPortFromSensor{&openPlopObject}
-
-	// Prepare indicators for FK
-	suite.NoError(suite.indicatorDataStore.AddProcessIndicators(
-		suite.hasWriteCtx, indicators...))
-
-	// Add PLOP referencing those indicators
-	suite.NoError(suite.datastore.AddProcessListeningOnPort(
-		suite.hasWriteCtx, fixtureconsts.Cluster1, plopObjects...))
-
-	suite.NoError(suite.indicatorDataStore.RemoveProcessIndicators(
-		suite.hasWriteCtx, indicatorIds))
-
-	_, err := suite.datastore.RemovePLOPsWithoutProcessIndicatorOrProcessInfo(suite.hasWriteCtx)
-	suite.NoError(err)
-
-	newPlopsFromDB := suite.getPlopsFromDB()
-	suite.Len(newPlopsFromDB, 0)
-}
+//// TestPLOPAddClosed: Happy path for ProcessListeningOnPort closing, one PLOP object is added
```

Comment:
Temporarily commenting out most unit tests. For now just worrying about the new SAC tests.

Reply:
No longer commenting out